apps/web/pages/signup.tsx

```diff
@@ -281,6 +281,7 @@ export default function Signup({
                 username={watch("username")}
                 premium={premiumUsername}
                 usernameTaken={usernameTaken}
+                disabled={!!orgSlug}
```

Comment:
Reverted to earlier implementation
![image](https://github.com/calcom/cal.com/assets/1780212/f86c4312-dee9-46ac-8608-fd702acc37b7)